lib/materials/data_helpers.rb

```diff
@@ -274,7 +274,7 @@ def materials_data( materials,
           unarchive_url: (material.is_a?(ExternalActivity) && policy(material).unarchive?) ? view_context.unarchive_external_activity_url(material) : nil,
           archive_url:   (material.is_a?(ExternalActivity) && policy(material).archive?) ? view_context.archive_external_activity_url(material) : nil,
           copy_url: external_copyable(material) ? view_context.copy_external_activity_url(material) : nil,
-          assign_to_class_url: current_visitor.portal_teacher && material.respond_to?(:offerings) ? "javascript:get_Assign_To_Class_Popup(#{material.id},'#{material.class.to_s}','#{t('material').pluralize.capitalize}')" : nil,
+          assign_to_class_url: current_visitor.portal_teacher && material.respond_to?(:offerings) ? "javascript:get_Assign_To_Class_Popup({material_id: #{material.id}, material_type: '#{material.class.to_s}', lightbox_material_text: '#{t('material').pluralize.capitalize}', skip_reload: true})" : nil,
```

Comment:
This seems to b changing the way the `get_Assign_To_Class_Popup is called.  It is passing an object instead of a list of arguments. The PR doesn't include changes to this javascript. Does the existing code already handle both ways of passing parameters?

Reply:
The change from a list of arguments to an object was made in a previous PR: https://github.com/concord-consortium/rigse/pull/721/commits/481a4a59e30171aca6e993418a91496153e459d1#diff-7d135b5f39ab1c66372ddbcf53545eb1R200

If I remember correctly, I overlooked this line in that other PR and corrected it here. Sorry that wasn't clear.